CHANGELOG.md

```diff
@@ -1,2 +1,5 @@
+## 2.0.2
```

Comment:
we don't have versions numbers in 2.x branch, as 2.0.0 is not released yet

Reply:
removed